examples/flow-counter/store/configureStore.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import { createStore, applyMiddleware } from 'redux';
+import type { Middleware, MiddlewareAPI, Dispatch, Store, ReduxAction } from 'redux';
+// import thunk from 'redux-thunk';
+import type { Thunk, ThunkMiddlewareDispatchable } from 'redux-thunk';
+import reducer from '../reducers';
+import type { AppState, Action } from '../types';
+
+// Re-defining redux-thunk here first to show how to type it, and also
+// because I don't know how to put the type in the declaration module.
+function thunk<State, Dispatchable: Object>(_ref: MiddlewareAPI<State, ThunkMiddlewareDispatchable<State, Dispatchable>>) : (next: Dispatch<Dispatchable>) => Dispatch<ThunkMiddlewareDispatchable<State, Dispatchable>> {
```

Comment:
Since I couldn't define the type for the default export of `redux-thunk`, I copy-pasted the code and _typed_ it here. The good thing is that we now have types for redux-thunk that we can then move to the repo.